_projects/mac_project.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: page_project
+title: Monitoring the Arctic Climate
+date: 2018-03-08
+updated: 2018-03-08
+navbar: Research
+subnavbar: Projects
+project_url:
+status:
+topics:
+  - apps
+keywords:
+  -
+head:
```

Comment:
Please provide the head of the project. Look at the [database](https://github.com/JLESC/jlesc.github.io/blob/source/_data/people.yml) of JLESC researchers to see if the person is already registered. If not, please do so.